### ISSUE TITLE
Nested includes work only for first key-value pair

### ISSUE THREAD

**Issue by funk-yourself** [vasilakisfil/SimpleAMS#41]
```ruby
[:some_association, another_association: [:nested]] # works
```

```ruby
[:some_association, another_association: [:nested], one_more: [:nested]] # one_more is not rendered
```

**Comment by vasilakisfil** [comment]
Unfortunately I won't have time to look into that until next week. Very curious though to find the bug though.

**Comment by vasilakisfil** [comment]
@funk-yourself which adapter are you using ?

**Comment by vasilakisfil** [comment]
@funk-yourself found the bug. I have merged into master #42 which should solve this issue. I haven't released a new version because travis takes ages (need to move over gh eventually) and because I want to add some more tests.

**Comment by vasilakisfil** [comment]
@funk-yourself published a new version (0.2.6) that includes among others this fix. Thanks for reporting!
